File: src/problems/leetcode-cn/171. Excel 表列序号/solution/java/Solution.java
class Solution {
    public int titleToNumber(String columnTitle) {
        byte[] arr = columnTitle.getBytes();
        int result = 0;
        for (int i = 0; i < arr.length; i++) {
            result = result * 26 + (arr[i] - 'A' + 1);
        }
        return result;
    }
}

File: src/problems/leetcode-cn/171. Excel 表列序号/solution/rust/solution.rs
impl Solution {
    pub fn title_to_number(column_title: String) -> i32 {
        let mut res = 0;
        for b in column_title.bytes() {
            res = res*26 + ((b - 'A' as u8) as i32 + 1);
        }
        res
    }
}